target/linux/ath79/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -71,4 +71,26 @@ case "$FIRMWARE" in
 		;;
 	esac
 	;;
+"ath9k-eeprom-pci-0000:00:11.0.bin")
+	case $board in
+	"dlink,dir-825-b1"|\
+	"netgear,wndr3800")
+		ath9k_eeprom_extract "caldata" 4096 3768
```

Comment:
The WNDR3800 uses "art" rather than "caldata".
In a recent commit quotes where removed ("ath79: remove quotes for boardname in scripts").